content/How an engineer can be your best partner.md

```diff
@@ -0,0 +1,86 @@
+# **GROWTH - Data Analysis based decisions**
+
+![main image](images/growth.jpg)
+
+Have you ever said one of these tricky phrases?
+  - I'm sure this feature is going to rock it!
+  - We need this feature before going live, it is a must!
+  - Users will use this for sure!
+
+Well, if you found yourself saying any of those without any numbers that support you, you may probably be making a lot of assumptions.
+So is like you are playing roulette, due to lacking data to prove your hypothesis.
+Give growth a try! And find out how to move smarter with your development team along with really learning what your users want!
+
+## 1) What's GROWTH
+
+I see growth as a way of working around data analysis based decisions. So first let's explain a bit what data analysis is:
+
+`Data analysis is defined as a process of cleaning, transforming, and modeling data to discover useful information for business decision-making. The purpose of Data Analysis is to extract useful information from data and taking the decision based upon the data analysis`
+
+You may see that `data` word came up a lot, that's not because I want to be redundant, instead is because I want to emphasize it as a really important concept. Nowadays the more data you have the more power you have to make better decisions.
```

Comment:
```suggestion
You may see that `data` word came up a lot, that's not because I want to be redundant but because I want to emphasize it as a really important concept. Nowadays the more data you have the more power you have to make better decisions.
```